src/containers/UserList/CardPerson.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { FC, useEffect, useState } from 'react';
+import Card from '@mui/material/Card';
```

Comment:
you can probably import whole material components by import {} from @mui/material